Add troubleshooting item for kubeadm cluster upgrade (#22792)
kubernetes/website: content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

Alternatively, you can modify `/etc/fstab` to make the `/usr` mount writeable, but please
be advised that this is modifying a design principle of the Linux distribution.

## `kubeadm upgrade plan` prints out `context deadline exceeded` error message

> This error message is shown when upgrading a Kubernetes cluster with `kubeadmin`in the case of running an external etcd. This is not a critical bug. You can proceed with `kubeadm upgrade apply ...`. 

Reviewer: please change:
>  cluster with `kubeadmin`

to:
> cluster with kubeadm

and:
> This is not a critical bug. 

to:
> This is not a critical bug and happens because older versions of kubeadm perform a version check on the external etcd cluster.

Author: Will do.

> This error message is shown when upgrading a Kubernetes cluster with `kubeadm`in the case of running an external etcd. This is not a critical bug and happens because older versions of kubeadm perform a version check on the external etcd cluster. You can proceed with `kubeadm upgrade apply ...`. 

Reviewer: missing space after `kubeadm`

> This issue has been fixed in version 1.19. 

Reviewer 2: ```suggestion
This issue is fixed as of version 1.19. 
```

Author: thanks for the review. Will update :)

> This issue has been fixed in version 1.19. So if you'd like to get rid of this error message, please upgrade your `kubeadmin` to version 1.19. 

Reviewer: let's remove the last sentence:
> So if you'd like to get rid of this error message, please upgrade your `kubeadmin` to version 1.19.

Author: will do.
